cmd/server/main.go

```diff
@@ -187,11 +198,37 @@ func runHandler(c *cli.Context) error {
 		return err
 	}
 
-	wsUrl := c.String("ws_url")
+	wsUrl := c.String("ws-url")
```

Comment:
Good catch! However, this flag is obviously not needed here since the service works 😊 This is only needed in cloud-ingress (where there is no such typo). In OSS ingress, we never pass in the wsUrl as part of the GetInfo response, and always read it from conf as it is constant for the deployment. We should probably remove this line instead, as well as the flag in the "run-handler" cli definition.